pkg/wcore/workspace.go

```
@@ -114,40 +115,76 @@ func UpdateWorkspace(ctx context.Context, workspaceId string, name string, icon
 	}
 
 	// delete all pinned and unpinned tabs
 	for _, tabId := range append(workspace.TabIds, workspace.PinnedTabIds...) {
 		log.Printf("deleting tab %s\n", tabId)
 		_, err := DeleteTab(ctx, workspaceId, tabId, false)
 		if err != nil {
-			return false, fmt.Errorf("error closing tab: %w", err)
+			return false, "", fmt.Errorf("error closing tab: %w", err)
 		}
 	}
 	windowId, err := wstore.DBFindWindowForWorkspaceId(ctx, workspaceId)
```

golangci-lint (1.62.2)

SA4006: this value of `err` is never used

(staticcheck)